packages/text-field/index.tsx

```diff
@@ -276,22 +272,13 @@ class TextField<T extends HTMLElement = HTMLInputElement> extends React.Componen
     };
   }
 
-  get helperTextAdapter(): Partial<MDCTextFieldAdapter> {
-    return {
-      showToScreenReader: () =>
-        this.setState({showHelperTextToScreenReader: true}),
-      setValidity: (isValid: boolean) => this.setState({isValid}),
-    };
-  }
-
   inputProps(child: React.ReactElement<InputProps<T>>) {
     // ref does exist on React.ReactElement<InputProps<T>>
     // @ts-ignore
     const {props, ref} = child;
     return Object.assign({}, props, {
       foundation: this.state.foundation,
       handleFocusChange: (isFocused: boolean) => this.setState({isFocused}),
-      handleValueChange: (value: string | number, cb: () => void) => this.setState({value}, cb),
```

Comment:
why was this removed?

Reply:
Input.tsx already has this method.
And I cannot find any special purpose of this function.
So I remove it.